eZ/Publish/API/Repository/Tests/ContentServiceTest.php

```diff
@@ -4819,4 +4819,33 @@ private function createFieldsFixture()
             ),
         );
     }
+
+    /**
+     * Test for the isContentTypeUsed() method.
+     *
+     * @see \eZ\Publish\API\Repository\ContentService::isContentTypeUsed()
+     * @group user
+     */
+    public function testIsContentTypeUsed()
+    {
+        if ($this->isVersion4()) {
+            $this->markTestSkipped('This test requires eZ Publish 5');
```

Comment:
i don't understand this.... since we're adding this to 6 wouldn't it required ```6``` instead of ```5```?